dynamic_listings/python/default/computer_science/dfs_inorder.py

```diff
@@ -1,14 +1,14 @@
-def dfs_traverse_inorder(n):
+def dfs_traverse_inorder(n: Node):
     # Step 1: We traverse the left subtree, using recursion
-    if n.left is not None:
+    if n.left:
         dfs_traverse_inorder(n.left)
     # Step 2: Visit the node, in this case we print its value
     print(n.content)
     # Step 3: We traverse the right subtre, using recursion
-    if n.right is not None:
+    if n.right:
         dfs_traverse_inorder(n.right)
```

Comment:
The same reasoning used in another review goes here, if Node contains either some (badly implemented) `__bool__` or `__len__` methods, the simplified check may fail in unexpected ways.